artifacts/scripts/util.sh

```diff
@@ -147,12 +149,12 @@ sync_repo() {
     fi
 
     # fetch upstream kube and checkout $src_branch, name it filtered-branch
-    git remote rm upstream-kube >/dev/null || true
-    git remote add upstream-kube "${kubernetes_remote}" >/dev/null
-    git fetch -q upstream-kube --no-tags
+    git remote rm upstream >/dev/null || true
+    git remote add upstream "${kubernetes_remote}" >/dev/null
+    git fetch -q upstream --no-tags
     git branch -D filtered-branch >/dev/null || true
-    git branch -f upstream-branch upstream-kube/"${src_branch}"
-    echo "Checked out k8s.io/kubernetes commit $(git rev-parse upstream-branch)."
+    git branch -f upstream-branch upstream/"${src_branch}"
+    echo "Checked out source commit $(git rev-parse upstream-branch)."
```

Comment:
our mixture of "source" and "upstream" is confusing. It's the same thing.

Reply:
yeah, it is hard to formulate a good message here

Reply:
Am fine with source or upstream, but we should choose one :)

Reply:
they I would leave it with "source" to make it consistent with the rest of the code

Reply:
:+1: